source/_static/js/main.js

```diff
@@ -30,3 +30,35 @@ window.addEventListener("DOMContentLoaded", (event) => {
 		$('body').toggleClass(target + '-toggled');
 	});
 });
+
+window.addEventListener('hashchange', function(){
+   // This hack ensures that if a URL hash is nested under a hidden tab,
+   // we "activate" that tab and display the proper contents.
+   //
+   // This lets us do nice things like use tabs to organize related
+   // configuration data.
+   //
+   // Few notes:
+   //
+   // sphinx-tabs uses some cool css to do this w/o any JS. But this structure
+   // doesn't use nesting, so we are entirely dependent on a certain ordering
+   // of elements:
+   //
+   // <radio id="foo"></radio>
+   // <label for="foo"</label>
+   // <div class="sd-tab-content"> Some stuff </div>
+   // <radio id="foo"></radio>
+   // <label for="foo"</label>
+   // <div class="sd-tab-content"> <p id="target">Foo</p></div>
+   //
+   // Getting to the parent div isn't too hard. But identifying the correct
+   // label to modify requires "looking" back one element and hoping
+   // the ordering is correct.
+
+   var parentPanel = document.getElementById(window.location.hash.substr(1)).closest(".sd-tab-content");
```

Comment:
This will mostly return null as the radio elements are placed in the same level as `.sd-tab-content`. Please check the below screenshot.

![Screenshot 2021-09-09 at 2 22 30 PM](https://user-images.githubusercontent.com/13393018/132655644-c13ac832-121b-454a-8f9f-53f6cf190c59.png)

As we already using jQuery, we can achieve the same with fewer codes as follows:

```
var hash = window.location.hash;
if (hash) {
   var target = $('.sd-tab-set ' + hash);  // .sd-tab-set is the parent wrapper of the tab group so adding it here in order to make sure the available ID is a tab
   if(target[0]) {
       target.prop('checked', true);
   }
}
```